data_steward/cdr_cleaner/cleaning_rules/remove_operational_pii_fields.py

```diff
@@ -0,0 +1,239 @@
+# coding=utf-8
+"""
+Removes operational Pii fields from rdr export.
+
+Some new operational fields exists, that were not blacklisted in the RDR export. These rows needs to be dropped in the 
+RDR load process so they do not make it to CDR. These do not have concept_id maps. 
+The supplemental operational_pii_fields.csv shows all present PPI codes without a mapped concepts,
+indicating which should be dropped in the “drop_value” column
+
+Jira issues = DC-500, DC-831
+"""
+# Python imports
+import logging
+import os
+from datetime import datetime
+
+# Project imports
+from google.cloud import bigquery
+from gcloud.bq import BigQueryClient
+import resources
+import constants.cdr_cleaner.clean_cdr as cdr_consts
+from cdr_cleaner.cleaning_rules.base_cleaning_rule import BaseCleaningRule, query_spec_list
+from common import JINJA_ENV, OBSERVATION
+from typing import Dict
+
+LOGGER = logging.getLogger(__name__)
+
+OPERATIONAL_PII_FIELDS_TABLE = '_operational_pii_fields'
+INTERMEDIARY_TABLE = 'remove_operational_pii_fields_observation'
```

Comment:
You could remove this constant and use a sandbox table name that is generated for you by the` def get_sandbox_tablenames(self)`